app/assets/javascripts/backbone/views/dashboard_calendar_view.js.coffee

```diff
@@ -0,0 +1,49 @@
+class MS.Views.DashboardCalendarView extends Backbone.View
```

Comment:
There is a fair amount of duplication here with the other calendar view. Perfect time to use inheritance. Move the methods that you want to use in both into the super class. Superclass (call it CalendarView?) extends Backbone.View. Then the child classes (TimelineCalendarView? and DashboardCalendarView) extend CalendarView.